### ISSUE TITLE
Allow several phone numbers in SMS action [ To be closed]

### ISSUE THREAD

**Issue by manucarrace** [telefonicaid/perseo-fe#337]
Allow several phone numbers in SMS action

**Comment by manucarrace** [comment]
This is an example of message received by acdc:
{"to":["tel:+34679245010"],"message":"Nivel de tanque AV_MEDVO_CA_0001 por debajo del umbral. Última medida 239","from":"tel:638444191;phone-context=+34"}

**Comment by AlvaroVega** [comment]
Yo mean that acdc would receive something like:
{"to":["tel:+34679245010", "tel:+34667766778"],"message":"Nivel de tanque AV_MEDVO_CA_0001 por debajo del umbral. Última medida 239","from":"tel:638444191;phone-context=+34"}

**Comment by AlvaroVega** [comment]
This new feature would imply modify "vrules".
```
iot-perseo-fe               | time=2019-05-24T08:13:32.919Z | lvl=INFO | corr=d0bb7621-5de8-497c-975a-f45b16c2d890; perseocep=79 | trans=d0bb7621-5de8-497c-975a-f45b16c2d890 | op=/m2m/vrules/sdfgsdfgsdfg | path=/m2m/vrules/sdfgsdfgsdfg | comp=perseo-fe | srv=smartcity | subsrv=/ | from=::ffff:172.17.0.15 | msg=incoming request {"method":"PUT","url":"/m2m/vrules/sdfgsdfgsdfg","headers":{"host":"iot-pep-perseo-fe:1026","connection":"close","user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0","accept":"application/json","accept-language":"en-US,en;q=0.5","accept-encoding":"gzip, deflate, br","referer":"https://localhost/visualrules/sdfgsdfgsdfg/view","content-type":"application/json;charset=utf-8","x-auth-token":"b119e06c909041ae83610635425904aa","fiware-service":"smartcity","fiware-servicepath":"/","cookie":"menuPanel=visible; menuPanelType=main; region3_registry_menu=visible; region1_identity_menu=none; region1_manage_menu=visible; region1_dashboard_main_menu=visible; region3_metadata_menu=none; userObligationsViewed=true; authToken=%7B%22XAuthToken%22%3A%22b119e06c909041ae83610635425904aa%22%2C%22domainName%22%3A%22smartcity%22%2C%22projectName%22%3Anull%2C%22extras%22%3A%7B%22password_creation_time%22%3A%222019-04-09T09%3A28%3A32Z%22%2C%22last_login_attempt_time%22%3A%222019-05-22T05%3A52%3A28.000000%22%2C%22pwd_user_in_blacklist%22%3Afalse%2C%22password_expiration_time%22%3A%222020-04-08T09%3A28%3A32Z%22%7D%7D","x-forwarded-for":"172.17.0.25","content-length":"572"},"body":{"name":"sdfgsdfgsdfg","active":1,"cards":[{"id":"card_2","type":"SensorCard","sensorCardType":"regexp","configData":{},"connectedTo":["card_3"],"conditionList":[{"scope":"XPATH","parameterValue":"aaaa","parameterName":"id","not":false,"operator":"MATCH","userProp":""}],"sensorData":{}},{"type":"ActionCard","actionData":{"userParams":[{"name":"sms","value":""},{"name":"sms.to","value":"+346666666 +347777777"},{"name":"sms.message","value":"Loreipsun message "}],"name":"SendSmsMibAction","type":"SendSmsMibAction"},"id":"card_3","connectedTo":[]}],"id":"sdfgsdfgsdfg"},"subservice":"/","service":"smartcity"}
```

**Comment by AlvaroVega** [comment]
Tested with a visual rule including 2 numbers in SMS action:
```
iot-perseo-fe               | time=2019-05-30T07:06:03.678Z | lvl=INFO | corr=5ea560c1-0370-43e9-913a-5452780e4740; perseocep=3 | trans=e472fe39-6705-40aa-b2d0-479d5c273dea | op=/actions/do | path=/actions/do | comp=perseo-fe | srv=smartcity | subsrv=/test | msg=smsAction.SendSMS done url:http://sms-endpoint/smsoutbound result:{"errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","hostname":"sms-endpoint","host":"sms-endpoint","port":80}, msg: {"to":["tel:+3412341234","tel:+346656566"],"message":"asdfasdfasdfasdf","from":"tel:22012;phone-context=+34"}
```

**Comment by fgalan** [comment]
Implemented in PR #341

**Comment by fgalan** [comment]
After the PR #341 merge, the following is yet pending to close this issue (https://github.com/telefonicaid/perseo-fe/pull/341#discussion_r288941035):

> Apart from this acceptance test (which I'm not sure is in use now), I'd suggest adding a component test. You can base in existing "should return ok with a valid action with a rule for sms", just adding a new case (a new JSON) which the new syntaxis for multi-SMS.
